app/models/user.rb

```diff
@@ -23,14 +23,13 @@ class User < ApplicationRecord
          :recoverable, :rememberable, :validatable
 
   has_many :tweets
+  has_many :visitors
```

Comment:
Are you using this association anywhere? is it really required?

Reply:
No, I only required it in tweet. I will fix it